app/controllers/wikis_controller.rb

```diff
@@ -42,11 +48,19 @@ def destroy
   private
 
   def wikis
-    @wikis ||= Wiki.all
+    if current_user.premium? || current_user.admin?
+      @wikis ||= Wiki.all
```

Comment:
Do you want a premium user to be able to see everyone's private wikis or just their own?